xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -30,6 +30,16 @@ @implementation TVOSLibInputTouch
 
 #pragma mark - gesture methods
 
+// called before any press or touch event
+- (BOOL)gestureRecognizer:(UIGestureRecognizer*)gestureRecognizer
```

Comment:
just in case please verify that all gestures have `delegate` set

Reply:
What do you mean? Not sure to understand.

Reply:
This is a delegate method (`UIGestureRecognizerDelegate`), it will be called only for those gesture recognisers that have `delegate` property set.